doc/about.rst

```diff
@@ -0,0 +1,123 @@
+.. _about:
+
+##############################################################################
+About
+##############################################################################
```

Comment:
Short headers is just my own preference, I was only mentioning them in case you weren't aware of that style (and also happen to like it). There's nothing wrong with the bold headers if you like them better.